src/kOS.Safe/Encapsulation/SuffixedList.cs

```diff
@@ -0,0 +1,21 @@
+using System.Collections.Generic;
```

Comment:
This class is the reason i did this list work. This will only let you make a list if its type is a structure that can be used by users. I would have made this the case everywhere but we do serve quite a few value types with lists and that wouldnt work.